cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -265,8 +269,9 @@ def test_suspend(
     Metadata:
         test_flag: power_control, provision
     """
-    testing_instance.wait_for_vm_state_change(
-        desired_state=testing_instance.STATE_ON, timeout=720, from_details=True)
+    if provider.type != 'azure':
```

Comment:
This will never get run against azure due to the uncollect on line 264, there is also a verify_vm_running fixture here, does that not do the job? If not, we should consider fixing that up instead.